spec/support/external_request.rb

```diff
@@ -0,0 +1,25 @@
+RSpec.configure do |config|
+  config.before(:suite) do
+    mock_request
+  end
+end
+
+def mock_request
```

Comment:
I'd name this more explicit, this does not only mock_request, this mocks the requests going to youtube / the commercial requests.